lib/Importer/ONNXIFIModelLoader.cpp

```diff
@@ -59,6 +61,30 @@ llvm::Expected<std::unique_ptr<ONNXIFIModelLoader>> ONNXIFIModelLoader::parse(
 
     // Keep hold of the context
     loader->core_ = std::move(onnxLoader);
+  } else {
+    // Use Caffe2 Model loader
```

Comment:
This is starting to seem a bit tedious and specific to each model type even though the underlying pattern I think is pretty consistent across model types being loaded. It would be better if possible to move the logic of loading each model type (stuff like calling `loadProto` `loadInputs`, `loadInitializers`, etc) to a `load()` function or something like that in `ONNXModelLoader` and `Caffe2ModelLoader` so we can just call `loader->core_.load(model, modelSize, weightsCount, ...)`. Maybe it could even be in the constructor of those underlying loaders those seem to already have a lot of the logic to do this. Is there anything preventing using those here instead?

Reply:
Yeah, I've mentioned that in the summary. We probably want to do it in a refacotring PR.